### PR TITLE
`ElevatedView` visual unification

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Feature
- 
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by jeromelaban** [comment]
We should now be able to validate the result in runtime tests using RenderTargetBitmap

File: src/Uno.UI.Toolkit/ElevatedView.cs [hunk 1]
```diff
@@ -194,9 +192,13 @@ private void UpdateElevation()
 #elif __IOS__ || __MACOS__
 				this.SetElevationInternal(Elevation, ShadowColor, _border.BoundsPath);
 #elif __ANDROID__
-				// The elevation must be applied on the border, since
-				// it will get the right shape (with rounded corners)
-				_border.SetElevationInternal(Elevation, ShadowColor);
+#if __ANDROID__
+				_invalidateShadow = true;
+				((ViewGroup)this).Invalidate();
+#endif
```
Comment (carldebilly) [inline src/Uno.UI.Toolkit/ElevatedView.cs:198]: Is it me or the `#if __ANDROID__` here seems useless? ;-)

File: src/Uno.UI.Toolkit/ElevatedView.cs [hunk 2]
```diff
@@ -212,6 +214,11 @@ private void UpdateElevation()
 
 		protected override Windows.Foundation.Size ArrangeOverride(Windows.Foundation.Size finalSize)
 		{
+#if __ANDROID__
+			_invalidateShadow = true;
+			((ViewGroup)this).Invalidate();
```
Comment (carldebilly) [inline src/Uno.UI.Toolkit/ElevatedView.cs:219]: ... and you're not causing a layout loop?

File: src/Uno.UI.Toolkit/UIElementExtensions.cs [hunk 3]
```diff
@@ -101,9 +101,9 @@ internal static void SetElevationInternal(this DependencyObject element, double
 				if (elevation > 0)
 				{
 					// Values for 1dp elevation according to https://material.io/guidelines/resources/shadows.html#shadows-illustrator
```
Comment (carldebilly) [inline src/Uno.UI.Toolkit/UIElementExtensions.cs:103]: This URL doesn't work anymore.

**Review by carldebilly — APPROVED** [review]

File: src/Uno.UI.Toolkit/ElevatedView.cs (hunk 2 above)
Comment (MartinZikmund) [reply src/Uno.UI.Toolkit/ElevatedView.cs:214]: This occurs only on ElevatedView property changes, and based on testing does not seem to cause issues... so far 😂

**Review by jeromelaban — APPROVED** [review]

**Comment by NVLudwig** [comment]
@MartinZikmund Here is what I found when comparing the screenshots to each other and to the Elevation Styles inside Figma.

The way this is being reported makes it difficult to QA the output. That said here are my observations:

1) Skia 1 dp elevation seems problematic but it could be the screenshot provided is to light compared to others. See below:
Skia1dp

2) Seems like there is an X-axis offset value on all elevations when there should not be any
X-axis

3) Most shadows seem darker, but again this could be the product of screenshots 
DarkerShadows

4) 24dp is missing in all screenshots

**Comment by MartinZikmund** [comment]
@NVLudwig thanks for the findings, I will try to tweak these in next PR, wanted to have this merged now though

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/4.5

**Pull request merged**